liteflow-labs/starter-kit
# Explore collections and search the collections

### Project organization

- Trello card: https://trello.com/c/Ixhvxhzl
- Trello card: https://trello.com/c/TrE2jJuD

### Description

Adding the Collections tab and its search functionality. I should be working the same the Users tab. 
We should be able to search name, address and description.

### How to test

Visit the explore page. Click on collections. You should be able to see the listing of collections, we currently only have 2 collections. I discussed with @EmmanuelDrouin about having some placeholder images for our demo collections.

If you want to test the pagination/results per page you can duplicated the array of the collections in the codebase but this is the exact same code as the users page.
